modules/category/category.service.js

```diff
@@ -1,25 +1,38 @@
 const Category = require('./category.model');
 
+const CATEGORY_NOT_FOUND = JSON.stringify([
+  {
+    lang: 'uk',
+    value: 'Категорій не знайдено',
+  },
+  {
+    lang: 'eng',
+    value: 'Category not found',
+  },
+]);
 class CategoryService {
-  getAllCategories() {
-    return Category.find();
+  async getAllCategories() {
+    const category = await Category.find();
+    return category;
   }
 
-  getCategoryById(id) {
-    return Category.findById(id);
+  async getCategoryById(id) {
+    return (await Category.findById(id)) || new Error(CATEGORY_NOT_FOUND);
   }
 
-  updateCategory(id, category) {
-    return Category.findByIdAndUpdate(id, category);
+  async updateCategory(id, category) {
+    return (
+      (await Category.findByIdAndUpdate(id, category))
```

Comment:
Please, add third parameter ` {new: true}` to return already updated entity

Reply:
fixed